slate-note-taking/src/Components/Context/NotetakingContext.js

```diff
@@ -19,8 +26,8 @@ function NotetakingContext({ children }) {
         return { ...state, inputTextTitleValue: action.payload };
       case "PRIORITYRADIOBOXVALUE":
         return { ...state, priorityRadioBoxValue: action.payload };
-      case "LABELRADIOBOXVALUE":
-        return { ...state, labelRadioBoxValue: action.payload };
+      case "LABELINPUTBOXVALUE":
```

Comment:
use underscores between two words for better readability like `LABEL_INPUT_BOX_VALUE`